### ISSUE TITLE
When hovering over objects, display an icon representing the action

### ISSUE THREAD

**Issue by mwcz** [CommandLineHeroes/hero-engine#44]
For instance if an object has `canlook`, hovering over it should display an eye icon (for example) next to the mouse cursor (not replacing the mouse cursor).

**Comment by Jared-Sprague** [comment]
This is a great start thanks @LiquidityC,  I think all that is left is making a set of icons for the following actions:
- Position change e.g. a door,  Icon could be two foot prints
- Inspect which is like a close look e.g. a window,  icon could be an eyeball
- Pickup,  icon could be like the old Baldurs gate pickup cursor example: 
![screenshot from 2018-12-06 13-40-03](https://user-images.githubusercontent.com/3926730/49604660-7b8dd400-f95c-11e8-9534-8f201aaef847.png)

**Comment by mwcz** [comment]
Yeah, I think so.  The Engine could provide default cursor icons but the games should definitely be able to override them.
